src/nxdoc/nuxeo-server/administration/configuration-templates.md

```diff
@@ -133,17 +133,18 @@ Templates are located in the "templates" directory (`$NUXEO_HOME/templates`). To
 
 Here are the templates provided by default:
 
-*   common: common template used by other templates;
-*   [default]({{page page='connecting-nuxeo-to-the-database'}}): default Nuxeo configuration template for test purpose;
-*   [https]({{page page='http-and-https-reverse-proxy-configuration'}}): (not recommended) template to make the server listen to port 443 (HTTPS);
-*   [postgresql]({{page page='postgresql'}}): PostgreSQL configuration template;
+*   `common`: common template used by other templates;
+*   [**`default`**]({{page page='connecting-nuxeo-to-the-database'}}): default Nuxeo configuration template for test purpose;
+*   [**`https`**]({{page page='http-and-https-reverse-proxy-configuration'}}): (not recommended) template to make the server listen to port 443 (HTTPS);
+*   [**`postgresql`**]({{page page='postgresql'}}): PostgreSQL configuration template;
 *   postgresql-quartz-cluster
-*   [mssql]({{page page='microsoft-sql-server'}}): MS SQL Server configuration template;
+*   [**`mssql`**]({{page page='microsoft-sql-server'}}): MS SQL Server configuration template;
 *   mssql-quartz-cluster
-*   [mysql]({{page page='mysql'}}): MySQL configuration template;
-*   [oracle]({{page page='oracle'}}): Oracle configuration template;
-*   oracle-quartz-cluster
-*   custom: sample custom templates. Of course, this template is empty by default. One should copy it outside `$NUXEO_HOME` and adapt to his needs. See related section below.
+*   [**`mysql`**]({{page page='mysql'}}): MySQL configuration template;
+*   [**`oracle`**]({{page page='oracle'}}): Oracle configuration template;
+*   [**`mariadb`**]({{page page='mariadb'}}): From FT 9.1+, MariaDB configuration template. For older versions of Nuxeo, the MySQL template should be used.
```

Comment:
You should put `mariadb` right after `mysql` as they are closely related.

I'd write

    MariaDB configuration template (since Nuxeo FT 9.1; for older versions of Nuxeo the `mysql` template should be used).